openfe/setup/ligand_network_planning.py

```diff
@@ -169,34 +176,27 @@ def generate_maximal_network(
       progress bar: if False, no progress bar will be shown. If True, use a
       tqdm progress bar that only appears after 1.5 seconds. You can also
       provide a custom progress bar wrapper as a callable.
+    n_processes: int
+        parallelization of network generation.
```

Comment:
Here and everywher eelse, could you:
1. adjust the indentation to two rather than four
2. add more details ("number of threads to parallelize network generation" or something a bit more explicit for our users)